README.md

```diff
@@ -163,6 +163,82 @@ import 'octane-components/dist/components/PaymentSubmission/PaymentSubmission.cs
 
 ![Screenshot of the PlanPicker component](./docs/payment-submission.png)
 
+## Actions
+
+In addition to components, `octane-components` provides access to "actions", or asynchronous interactions with our API. While these complement our React components nicely, they can be used in any client-side app. They all return Promises and work well with `await`, when available.
+
+### `getActiveSubscription(token): Promise<PricePlan | null>`
+
+`hasPaymentInfo` is a simple action that checks for a customer's active subscription. If it's there, it resolves to the PricePlan details for that subscription. If the customer has no subscription, it resolves to `null`.
+
+**Example**
+
+```js
```

Comment:
asking for myself - is this the proper way to declare JS code blocks in MD?

Reply:
Yup — at least in Github-flavored markdown, adding a syntax identifier to a codeblock will add syntax highlighting. I think you can also do `javascript`, for example.